pwm-fan.sh

```diff
@@ -106,7 +106,7 @@ export_pwmchip_channel () {
 
 fan_initialization () {
 	if [[ -z "$TIME_STARTUP" ]]; then
-		TIME_STARTUP=60
+		TIME_STARTUP=10
```

Comment:
why was the default startup time reduced from 60 to 10s?

Reply:
Hello @cgomesu, thanks for your prompt feedback !
I have tested the changes on **Armbian 21.08.6 Bullseye** with **Linux 5.10.63-rockchip64**.
I have tested with all default parameters and with the following custom parameters: `-t 45 -d 25 -T 85 -D 100 -u 45 -U 55`, monitoring the temperature of the CPU with htop.
I found the fan control behaves as expected in both situations. I am currently running the fan control as a service with the mentioned parameters and haven't notice anything suspicious since yesterday.

Armbian for the NanoPi M4v2 comes with your script set with TIME_STARTUP=10, this is why I changed it. I can revert this to 60s for the PR. I must look into the rest of your comments and then propose a patch. :)

Reply:
okay.  the change to the startup time sounds like a different issue than the open being addressed by this PR.  after this one, you can submit another PR to make such a change if there is a justification for it.  so, for now, please keep the original one, namely `TIME_STARTUP=60`.